install/Knative-with-GiantSwarm.md

```diff
@@ -0,0 +1,184 @@
+tar xzf gsctl-0.14.1-linux-amd64.tar.gz
+sudo cp gsctl-0.14.1-linux-amd64/gsctl /usr/local/bin/
+```
+
+### Installing kubectl
+
+1.  If you already have `kubectl` CLI, run `kubectl version --short` to check
+    the version. You need v1.10 or newer. If your `kubectl` is older, follow the
+    next step to install a newer version.
+
+2.  [Install the kubectl CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl).
+
+## Cluster Setup
+
+Now that we have all the tools, we need a Kubernetes cluster to install Knative.
+
+### Configure your Giant Swarm installation
+
+First let's identify your Giant Swarm installation and save it for use later.
+
+1. Run `gsctl login <user@example.com> --endpoint <api.example.com>` replacing with your email account and your installation API endpoint.
+1. Test your gsctl connection:
+    ```bash
+    gsctl ping
+    ```
+
+### Create a Kubernetes cluster with Giant Swarm
+
+Next we will create a managed Kubernetes cluster in your Giant Swarm organisation. To make sure the cluster is large enough to host all the Knative and Istio components, the recommended configuration for a cluster is:
+
+* Kubernetes version 1.10 or later
+* Three or more nodes
+* RBAC is enabled by default
+
+1. Create the Giant Swarm cluster
+    ```bash
+    gsctl create cluster \
+        --name 'Knative cluster' \
+        --owner my-organisation \
+        --num-workers 3
+    ```
+
+1. Get id of your created cluster
+    ```bash
+    gsctl list clusters
+
+    ID     ORGANIZATION       NAME                                   RELEASE  CREATED
+    2edvr  my-organisation    Knative cluster                        2.0.1    2018 Nov 15, 15:06 UTC
+    ```
+
+1. Configure kubectl to use the new cluster.
+    ```bash
+    gsctl create kubeconfig --cluster=2edvr --certificate-organizations='system:masters'
+    ```
+
+    > As RBAC is configured by default on the cluster, you have to be part of `system:masters` to be admin on the cluster.
+
+1. Verify your cluster is up and running
+    ```bash
+    kubectl get nodes
+    ```
+
+## Installing Istio
+
+Knative depends on Istio.
+
+1. Install Istio:
+    ```bash
+    kubectl apply --filename https://raw.githubusercontent.com/knative/serving/v0.2.1/third_party/istio-1.0.2/istio.yaml
+    ```
+1. Label the default namespace with `istio-injection=enabled`:
+    ```bash
+    kubectl label namespace default istio-injection=enabled
+    ```
+
+1. Monitor the Istio components until all of the components show a `STATUS` of
+`Running` or `Completed`:
+    ```bash
+    kubectl get pods --namespace istio-system
+    ```
+
+It will take a few minutes for all the components to be up and running; you can
+rerun the command to see the current status.
+
+> Note: Instead of rerunning the command, you can add `--watch` to the above
+  command to view the component's status updates in real time. Use CTRL + C to exit watch mode.
+
+## Installing Knative components
```

Comment:
The "Installing Knative" section looks to be a version behind. Current install guides look similar to this one:

https://github.com/knative/docs/blob/master/install/Knative-with-GKE.md

We have moved away from bundle installs and are now recommending that each component be installed separately. Please update this section to align with the current (0.3) install guides.